csrankings-c.csv

```diff
@@ -585,7 +586,7 @@ Chiehyeon Lim,UNIST,http://service.unist.ac.kr,NW7F08MAAAAJ
 Chien-Chao Tseng,National Chiao Tung University,https://win.cs.nctu.edu.tw/cctseng,ownGePIAAAAJ
 Chien-Chung Shen,University of Delaware,https://www.eecis.udel.edu/~cshen,QyipVRoAAAAJ
 Chien-Ju Ho,Washington University in St. Louis,http://chienjuho.com,DWKoeW0AAAAJ
-Chien-Ming Huang,Johns Hopkins University,https://www.cs.jhu.edu/~cmhuang,0kTel90AAAAJ
+Chien-Ming Huang 0001,Johns Hopkins University,https://www.cs.jhu.edu/~cmhuang,0kTel90AAAAJ

```

Comment:
Not sure why CI is throwing error here: 
```
 Invalid name (Chien-Ming Huang 0001). Please ensure it matches the DBLP entry.
```
I can see the name disambiguated with "0001": https://dblp.org/pid/56/3778-1.html